src/lib/provable/gadgets/foreign-field.ts

```diff
@@ -432,11 +432,7 @@ function equals(x: Field3, c: bigint, f: bigint) {
   }
 }
 
-const provableLimb = modifiedField({
-  toInput(x) {
-    return { packed: [[x, Number(l)]] };
-  },
-});
+const provableLimb = modifiedField({});
```

Comment:
having a `toInput` method but no `check` method is a major footgun that caused the ecdsa vulnerability, because fileds are packed but unpacking is not constrained.
Since we don't want `check()` here I'm removing `toInput`.

Reply:
Dang! Sneaky footgun. Are there other potential areas this is occurring? 

Should we also update the `provable` interface or somewhere relevant to mention the case of leaving out certain provable methods?

Reply:
I did add docs on `Hashed` and `Packed` specifically about this issue. Other than that, I think it's fine because this is more low-level than users typically operate in.